onchain/rollups/.changeset/six-doors-grow.md

```diff
@@ -0,0 +1,8 @@
+---
+"@cartesi/rollups": major
+---
+
+Modified the IInputBox interface:
+
+-   removed `sender` from `InputAdded` event.
+-   correct the naming of `input` and `payload`.
```

Comment:
Renaming is not very important for positional parameters. Solidity doesn't support keyword arguments like Python, so we don't have to worry. They are backwards-compatible. Meanwhile, semantic changes have to be notified.

```suggestion
-   Modified the `InputAdded` event:

    -   Removed the `sender` parameter.
    -   Changed the semantics of the `input` parameter.

-   Added an `InputTooLarge` error. 
```